src/app/state/card/card.selectors.ts

```diff
@@ -68,9 +68,46 @@ export const selectCardsPageInfo = createSelector(
     pageIndex: cardState.pageIndex,
   }),
 );
+export const selectCardsSorted = createSelector(
+  selectAllCards,
+  selectCardState,
+  (allCards, cardState) => {
+    const getLastName = (obj: Card) => obj?.name?.split(' ')?.pop() || obj.name;
+    const sortedCards = allCards.sort((a: Card, b: Card) => {
```

Comment:
Ich würde hier noch eine Kopie des Arrays machen und dann sortieren. Da Array.sort() in place ist.